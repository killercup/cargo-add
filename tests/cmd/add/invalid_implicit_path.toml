bin.name = "cargo-add"
args = ["add", "./tests/fixtures/local"]
status.code = 1
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
