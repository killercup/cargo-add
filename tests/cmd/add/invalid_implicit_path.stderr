Command failed due to unhandled error: Unable to open local Cargo.toml

Caused by: Failed to read manifest contents
Caused by: [..]
